frame/contracts/src/weights.rs

```diff
@@ -155,1810 +150,1502 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
-			// Standard Error: 16_000
-			.saturating_add((27_785_000 as Weight).saturating_mul(r as Weight))
+		(51_490_000 as Weight)
+			// Standard Error: 71_000
+			.saturating_add((27_683_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_br_table_per_entry(e: u32, ) -> Weight {
-		(86_048_000 as Weight)
-			// Standard Error: 1_000
-			.saturating_add((82_000 as Weight).saturating_mul(e as Weight))
+		(77_260_000 as Weight)
+			// Standard Error: 2_000
+			.saturating_add((130_000 as Weight).saturating_mul(e as Weight))
 	}
 	fn instr_call(r: u32, ) -> Weight {
-		(54_654_000 as Weight)
-			// Standard Error: 82_000
-			.saturating_add((199_159_000 as Weight).saturating_mul(r as Weight))
+		(52_012_000 as Weight)
+			// Standard Error: 564_000
+			.saturating_add((188_018_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_call_indirect(r: u32, ) -> Weight {
-		(67_478_000 as Weight)
-			// Standard Error: 113_000
-			.saturating_add((302_597_000 as Weight).saturating_mul(r as Weight))
+		(65_670_000 as Weight)
+			// Standard Error: 5_489_000
+			.saturating_add((294_560_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_call_indirect_per_param(p: u32, ) -> Weight {
-		(384_281_000 as Weight)
-			// Standard Error: 13_000
-			.saturating_add((9_984_000 as Weight).saturating_mul(p as Weight))
+		(368_428_000 as Weight)
+			// Standard Error: 26_000
+			.saturating_add((10_469_000 as Weight).saturating_mul(p as Weight))
 	}
 	fn instr_local_get(r: u32, ) -> Weight {
-		(55_473_000 as Weight)
-			// Standard Error: 16_000
-			.saturating_add((9_287_000 as Weight).saturating_mul(r as Weight))
+		(52_091_000 as Weight)
+			// Standard Error: 32_000
+			.saturating_add((11_160_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_local_set(r: u32, ) -> Weight {
-		(55_426_000 as Weight)
-			// Standard Error: 38_000
-			.saturating_add((10_559_000 as Weight).saturating_mul(r as Weight))
+		(52_145_000 as Weight)
+			// Standard Error: 18_000
+			.saturating_add((12_086_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_local_tee(r: u32, ) -> Weight {
-		(55_332_000 as Weight)
-			// Standard Error: 8_000
-			.saturating_add((15_640_000 as Weight).saturating_mul(r as Weight))
+		(52_057_000 as Weight)
+			// Standard Error: 26_000
+			.saturating_add((2_555_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_global_get(r: u32, ) -> Weight {
-		(74_497_000 as Weight)
-			// Standard Error: 22_000
-			.saturating_add((15_067_000 as Weight).saturating_mul(r as Weight))
+		(73_126_000 as Weight)
+			// Standard Error: 35_000
+			.saturating_add((16_004_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_global_set(r: u32, ) -> Weight {
-		(74_445_000 as Weight)
-			// Standard Error: 49_000
-			.saturating_add((17_650_000 as Weight).saturating_mul(r as Weight))
+		(73_104_000 as Weight)
+			// Standard Error: 63_000
+			.saturating_add((2_267_000 as Weight).saturating_mul(r as Weight))
```

Comment:
It seems like each time we run the benchmark different functions drop to very low weight randomly. I don't get it. The code wasn't changed in-between.

Reply:
The bespoken follow up PR: https://github.com/paritytech/substrate/pull/9712